src/dbmanager/dbmanager_test.go

```diff
@@ -37,12 +38,12 @@ func TestRouteID(t *testing.T) {
 	db, err := dbOpen()
 	require.NoError(t, err)
 	dbmanager := NewDBManager(db)
-	id1, err := dbmanager.insertRoute(7, 32, 44, 1500, "2019-02-24 08:30:00")
+	id1, err := dbmanager.insertRoute(context.Background(), 7, 32, 44, 1500, "2019-02-24 08:30:00")
 	require.NoError(t, err)
-	_, err = dbmanager.insertRoute(7, 32, 44, 1520, "02-24 08:30:00")
+	_, err = dbmanager.insertRoute(context.Background(), 7, 32, 44, 1520, "02-24 08:30:00")
```

Comment:
You can write `ctx :=context.Background()` and then use `ctx` as param instead all `context.Background()`

Reply:
Also you can add timeout to context to control time of test`s execution.